sdk/go/integration_test.go

```diff
@@ -17,14 +17,18 @@ const spinBinary = "../../target/debug/spin"
 func retryGet(t *testing.T, url string) *http.Response {
 	t.Helper()
 
-	const maxTries = 120 // (10min/5sec)
+	const maxTries = 600 // (10min)
```

Comment:
Why do we try for so long?

Reply:
:shrug:

https://github.com/fermyon/spin/blob/3f7c2fee0c22fe24ea4fc74fd516142b02936cce/sdk/go/integration_test.go#L44-L45

Reply:
The timeout is long because we need to wait for the build also.  TinyGo can take a bit of time in CI